webpack-and-babel/imports.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable no-unused-vars */
+
+// import from a file located at the absolute path `/foo/bar/baz/qux.js`:
+// eslint-disable-next-line import/no-absolute-path
+import f1 from '/foo/bar/baz/qux';
+
+// import from a file named `foo.json` located in the current directory:
+import f2 from './foo';
```

Comment:
In this case we want to include the file extension. We can drop the file extension if it's `js, jsx, ts, tsx`. Otherwise it's going to try and default to `foo.js` which won't exist